modules/channel-telegram/src/renderers/image.ts

```diff
@@ -0,0 +1,31 @@
+import * as sdk from 'botpress/sdk'
+import { TelegramContext } from 'src/backend/typings'
+
+export class TelegramImageRenderer implements sdk.ChannelRenderer<TelegramContext> {
+  getChannel(): string {
+    return 'telegram'
+  }
+
+  getPriority(): number {
+    return 0
+  }
+
+  getId() {
+    return TelegramImageRenderer.name
+  }
+
+  async handles(context: TelegramContext): Promise<boolean> {
+    return context.payload.type === 'image'
+  }
+
+  async render(context: TelegramContext): Promise<void> {
+    const { messages } = context
+    const payload = context.payload as sdk.ImageContent
+
+    if (payload.image.toLowerCase().endsWith('.gif')) {
+      messages.push({ animation: payload.image })
+    } else {
+      messages.push({ photo: payload.image })
```

Comment:
I prefer image to photo, but I imagine this is a telegram thing

Reply:
Yes as I've said in another comment, what we are adding in messages is the payload that telegram takes. So here we are using telegram's language with `photo`